org.flatpak.tauno-serial-plotter.yml

```diff
@@ -0,0 +1,50 @@
+app-id: org.flatpak.tauno-serial-plotter
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: tauno-serial-plotter.py
+
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=network
+  - --device=all
+
+modules:
+  - python3-requirements.json
+
+  - name: tauno-serial-plotter
+    buildsystem: simple
+    build-commands:
+      - install -D src/tauno-serial-plotter.py /app/bin/tauno-serial-plotter.py
+      - install -d /app/share/applications
+      - install -m644 Tauno-Serial-Plotter.desktop /app/share/applications/Tauno-Serial-Plotter.desktop
```

Comment:
```suggestion
      - install -Dm644 Tauno-Serial-Plotter.desktop /app/share/applications/Tauno-Serial-Plotter.desktop
```

Reply:
you aslo need to add a `rename-desktop-file: Tauno-Serial-Plotter.desktop` around line 6.